resources/results.py

```diff
@@ -80,11 +80,11 @@ def get_clumping_results_path(pop: str = 'full', high_quality: bool = False,
     mt_name = 'max_pops' if max_pops else (f'{"not_" if not_pop else ""}{pop}' if pop in POPS else 'full_clump_results')
     
     if high_quality:
-        return f'{public_bucket}/clump_results_high_quality/{mt_name}.mt'
-    elif hq_phenos:
-        return f'{bucket}/ld_prune/clump_results_hq_phenos/{mt_name}.mt'
+        return f'{ldprune_dir}/clump_results_high_quality_22115/{mt_name}.mt' 
+    elif not high_quality and hq_phenos:
```

Comment:
`not high_quality` is implied here by the else. Should it be different if `high_quality and hq_phenos` (compared to `high_quality and not hq_phenos`)? Fine if not but just wanted to double check